### ISSUE TITLE
Feature: Run it on Kubernetes

### ISSUE THREAD

**Issue by adelerhof** [ckabalan/visualsubnetcalc#19]
I run this app on Kubernetes, interested in the manifests?

**Comment by ckabalan** [comment]
Yes please share them here, I'll look at testing and incorporating them into the documentation.

Can I ask why you self-host instead of use the website? I understand the desire to self-host for security or even just tinker/hobby. Please let me know if there is a specific concern that others might share that I could mitigate with more transparency or adjustment in approach.

**Comment by adelerhof** [comment]
First of all, thanks for this tool. I'll add the resources to a PR.
The reason I'm running a couple of tools on my Kubernetes cluster are multiple, for example:

- Sometimes other websites go down
- I have an Kubernetes cluster for testing/playing/experience/hobby, an empty cluster is of no use off course, so I'm always on the lookout for interesting tools.
- I want to use the latest images, sometimes they lag behind.
- Sometimes I forget where that handy tool was located, yes I know I can use favorites in browsers.
- Because I can is always a good reason for me ;)

**Comment by adelerhof** [comment]
See https://github.com/ckabalan/visualsubnetcalc/pull/23
